frontend/src/pages/projects/screens/detail/overview/serverModels/NoProjectServingEnabledSection.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { CardBody, FlexItem, Text, TextContent } from '@patternfly/react-core';
+import { ProjectObjectType, SectionType } from '~/concepts/utils';
+import OverviewCard from '~/pages/projects/screens/detail/overview/components/OverviewCard';
+import CollapsibleSection from '~/pages/projects/screens/detail/overview/components/CollapsibleSection';
+
+const NoProjectServingEnabledSection: React.FC = () => (
+  <CollapsibleSection sectionType={SectionType.training} title="Serve models">
+    <FlexItem>
+      <OverviewCard
+        objectType={ProjectObjectType.modelServer}
+        sectionType={SectionType.serving}
+        title="No deployed models"
```

Comment:
There are tests expecting this text to be "No model serving platform selected", @DaoDaoNoCode believes this code should be updated to match the test and the old text for this space. cc @lucferbux 

```suggestion
        title="No model serving platform selected"
```

Reply:
Fixed.